src/transformers/__init__.py

```diff
@@ -4330,6 +4340,9 @@
         TokenSpan,
     )
 
+    # Tools
+    from .tools import Agent, HfAgent, OpenAiAgent, PipelineTool, RemoteTool, Tool, launch_gradio_demo, load_tool
```

Comment:
```suggestion
    from .tools import HfAgent, OpenAiAgent, PipelineTool, RemoteTool, launch_gradio_demo, load_tool
```

Reply:
To limit breaking / design changes maybe it might be good to not put too much in the public API?